docker-compose.yml

```diff
@@ -18,6 +18,12 @@ services:
       - ${PWD}:/app
       - /var/run/docker.sock:/var/run/docker.sock
     network_mode: "host"
+    healthcheck:
```

Comment:
This will always fail due to no web service running on localhost (as defined in the same compose)